provisioning/transport/http/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "azure-iot-provisioning-http",
+  "version": "1.0.0",
+  "description": "HTTP transport for Azure device provisoning SDK",
```

Comment:
>device provisoning [](start = 43, length = 18)

needs to be double checked but if this is an official service name it probably needs to be capitalized.